### DISCUSSION TITLE
Select does not have a size property

### DISCUSSION THREAD

**Discussion by iessa-pragg-ctct** [primefaces/community#2381] [PrimeVue]
Select is a bit inconsistent with the other input fields as it does not have a `size` property. Would be nice to be consistent here so I don't have to manually style this particular control differently to others.

Though perhaps `InputText` is the inconsistent one for having a `size` property 🤷 

Currently just working around this with some CSS, but its not very resilient to changes.
```
*[size='small'] .p-select-label {
  font-size: var(--p-inputtext-sm-font-size);
  padding: var(--p-inputtext-sm-padding-y) var(--p-inputtext-sm-padding-x);
}

```

**Comment by BenJackGill** [comment]
There are los of discussions around this but no traction on it yet. In an attempt to get more visibility to the problem here are some of the other discussions asking for a `size` prop on all form components:

- https://github.com/orgs/primefaces/discussions/2493
- https://github.com/orgs/primefaces/discussions/2684
- https://github.com/orgs/primefaces/discussions/2381
- https://github.com/orgs/primefaces/discussions/2809

I love PrimeVue! So hoping they can add this soon.
